components/fonts/platform/windows/font_list.rs

```diff
@@ -25,7 +25,7 @@ where
 {
     let system_fc = FontCollection::system();
     for family in system_fc.families_iter() {
-        callback(family.name());
+        callback(family.family_name().unwrap());
```

Comment:
Instead of using unwrap, please handle the error:

```suggestion
        if Ok(family_name) = family.family_name() {
            callback(family_name);
        }
```